### ISSUE TITLE
Bimodal NTRs with in flight switching

### ISSUE THREAD

**Issue by lurkoholic** [Raptor831/RFStockalike#40]
The bimodal NTRs like the Atomic Age LANTERN & the LV-N Trimodal have "Switch Engine Mode" and "Toggle Mode" buttons in flight. This can allow more than one active mode at a time. I think there are *four* total engine modes instead of two.

[Screenshots] (http://imgur.com/a/VR0Kz)

**Comment by Raptor831** [comment]
Yeah, definitely has two engine modules active. I'll have to look into that one.

**Comment by lurkoholic** [comment]
They're both outdated, and I haven't tried creating a new one using the webapp.
